src/desktop/assets/main_layout.coffee

```diff
@@ -3,4 +3,4 @@ CurrentUser = require '../models/current_user.coffee'
 
 $ ->
   require('../components/main_layout/client.coffee')()
-  # require('../apps/auth2/client').open()
+  require('../apps/auth2/client').initModalManager()
```

Comment:
We might want to consider moving the modal manager init call to the [global_client_setup file](https://github.com/artsy/force/blob/master/src/desktop/lib/global_client_setup.coffee) so that it's available globally.

Reply:
Does the `init` fn in `auth2/client/index.js` still need to be `export`ed? 

Agree about moving to global client 👍 One thing we'd need to be sure about is that the mount #ID is always available no matter what kind of layout is being used

Reply:
regarding the `init` fn, yes we'll still need it [here](https://github.com/xtina-starr/force/blob/auth-system/src/desktop/assets/auth2.coffee) to support the static login page.

I get why we need it always available otherwise we can't render the  auth modal. What I don't know about is how we would ensure it's always available. Could you elaborate about that?

Reply:
Sure thing -- there are a few layout templates that we use from [here](https://github.com/artsy/force/tree/master/src/desktop/components/main_layout/templates) (as well as on the mobile side) and I _believe_ there are a few stray templates in places like `apps/article`. If we're going to centralize the login we need to be sure that all of these locations are accounted for. 

Alternatively -- and this might be better since everything would be dynamic and wouldn't need to litter layout code with #ids -- we could use JS and create a DOM node on the fly that the modal then attaches to, and on modal close it removes itself from the DOM.